src/screens/sync/index.tsx

```diff
@@ -35,41 +27,22 @@ const styles = StyleSheet.create({
 });
 
 export const SyncScreen: ScreenComponent<"Sync"> = ({ route }) => {
+  const { role } = route.params;
   const [viewMode, setViewMode] = React.useState<ViewMode>("list");
-  const [modalContent, setModalContent] = React.useState<TitleAndDescription>({
-    title: "",
-    description: "",
-  });
-  const { sheetRef, openSheet } = useBottomSheetModal({ openOnMount: false });
-
-  function setAndOpenModal(titleAndDescription: TitleAndDescription) {
-    setModalContent(titleAndDescription);
-    openSheet();
-  }
-
   return (
-    <React.Fragment>
-      <ScrollView
-        style={styles.container}
-        contentContainerStyle={styles.scrollContentContainer}
-      >
-        <ProjectInfo
-          name="Project Catapult"
-          viewMode={viewMode}
-          toggleViewMode={() => {
-            setViewMode((prev) => (prev === "list" ? "bubbles" : "list"));
-          }}
-        />
-        <Devices>
-          {viewMode === "list" ? (
-            <DeviceList setAndOpenModal={setAndOpenModal} />
-          ) : null}
-        </Devices>
-      </ScrollView>
-      <BottomSheetModal ref={sheetRef}>
-        <SyncGroupBottomSheetContent content={modalContent} />
-      </BottomSheetModal>
```

Comment:
Adding the bottoms sheet to `<Devices/>` can cause the modal to be at the bottom of the scroll view, which may be outside of the screen

Reply:
do you have an example of this issue? i'm not seeing an issue on my device related to the positioning of the sheet (even after scrolling down the list)

Reply:
the positioning of the modal is kind of independent of where in the "DOM" tree it's placed based on how the bottom sheet dep works. its positioning is usually adjusted via props you provide to the bottom sheet component, which we don't do at all in this case

Reply:
maybe this was an unrelated issue, but when i had a short list, my modal was not showing up in the right place (like it was showing up with a gap between the bottom of the screen)...but maybe that was unrelated to scroll view, because it seems to be working....

Reply:
yeah i think i fixed some issues with the bottom sheet in this PR - so possible that it was addressed here somewhere v4 vs v2 behavior for the dep is pretty different so had to make adjustments from Mapeo Mobile's implementation 😅